### PR TITLE
add aria-label to input element

### PR DESCRIPTION
**Pull request by borntofrappe**
- **What does this PR do?**

  Adds `aria-label` attribute to input elements, as discussed in issue #102

### CODE REVIEW

**Review by apollonian — COMMENTED** [review]
@borntofrappe Thank you for opening the PR!
I was wondering if we should have more descriptive label for the first input since this text would be read aloud by the assistive tech. For e.g., the first input can have the aria-label: "Please enter the verification code digit/character (depending on isInputNum) 1", and the rest of them can have "Digit/character n".

**Comment by borntofrappe** [comment]
@apollonian looking at the structure of the existing HTML, I would advise against such a label. As the assistive technology reaches the `<form>` element, the string "Enter verification code" should be enough to describe the purpose of the `<input>` elements which follow. To put more emphasis on the instruction, it's perhaps better to change the paragraph element into a heading.

What do you think?

**Comment by apollonian** [comment]
In that case, having a prop that allows changing the aria-label would come handy! The "Enter verification code" heading is in place for demo but doesn't actually ship with the component itself. See the CodeSandbox for reference: https://codesandbox.io/s/react-otp-input-demo-v2-1iy52

**Comment by borntofrappe** [comment]
I see. I notice that in the demo, the `<input>` elements are also wrapped in a `<div>` container. Considering the render logic of the `<OtpInput>` component (line 316 of [lib/index.jsx](https://github.com/devfolioco/react-otp-input/blob/master/src/lib/index.jsx)), what do you think about using a `<form>` element instead?

```jsx
<form
        style={Object.assign(
          { display: 'flex' },
          isStyleObject(containerStyle) && containerStyle
        )}
        className={!isStyleObject(containerStyle) ? containerStyle : ''}
      >
        {this.renderInputs()}
</form>
```

I say this because the `<form>` element has semantic meaning, unlike the general `<div>` element. In this way you can add a label on the form container instead of modifying the label of the first `<input>`.

```jsx
<form aria-label="Enter verification code">
</form>
```

Let me know what you think. I will accommodate whatever you decide :)

**Comment by apollonian** [comment]
@borntofrappe No worries. Thank you for all the suggestions! We do actually need the aria-label to be present on the `input` elements. https://wave.webaim.org/report#/https://devfolioco.github.io/react-otp-input 

At the moment, having a more descriptive label for the first input sounds like the right way ahead to me. If an aria-label is passed via props then we may override it for all the inputs.

**Comment by borntofrappe** [comment]
Understood, I will go ahead and modify the aria-label of the input elements as suggested.

File: src/lib/index.jsx [hunk 1]
```diff
@@ -90,7 +91,7 @@ class SingleOtpInput extends PureComponent<*> {
     return (
       <div className={className} style={{ display: 'flex', alignItems: 'center' }}>
         <input
-          aria-label={`Verification code ${isInputNum ? 'digit' : 'character'}`}
+          aria-label={`${(index === 0) ? 'Please enter verification code. ' : ''}${isInputNum ? 'Digit' : 'Character'} n° ${index + 1}`}
```
Comment (apollonian) [inline src/lib/index.jsx:94]: @borntofrappe Just one final thing. "n°" won't be read as "number" I believe. Should we probably just remove it?

Comment (borntofrappe) [reply src/lib/index.jsx:94]: Would you rather have it say "number"?

Comment (apollonian) [reply src/lib/index.jsx:94]: Yeah, I did consider "number" but then realised it might sound confusing incase of a digit: "Digit number one".

**Review by apollonian — APPROVED** [review]
Thanks @borntofrappe! 🎉

**Comment by apollonian** [comment]
@allcontributors please add @borntofrappe for code, a11y

**Pull request merged**